migrations/versions/098cb17abfbd_create_coins_table.py

```diff
@@ -0,0 +1,27 @@
+"""create coins table
+
+Revision ID: 098cb17abfbd
+Revises: 3b155acc6435
+Create Date: 2020-05-28 19:59:10.507974
+
+"""
+import sys
+sys.path.append('/app')
+
+import sqlalchemy as sa
+
+from models import coins
+
+# revision identifiers, used by Alembic.
+revision = '098cb17abfbd'
+down_revision = '3b155acc6435'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    coins.create(sa.engine)
```

Comment:
That's not how I really recommend it. Migrations should be applied repeatedly regardless of the table schema.

Reply:
https://github.com/fincubator/gateways_api/blob/4d0c807a1f8859092da60e06dab3b7017ddf3003/migrations/versions/3b155acc6435_create_assets_table.py#L23